package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.79.7",
+  "version": "2.79.6-breathing-data-not-being-persisted",
```

Comment:
@kuda1992 - minor point, but consider renaming the suffix as `breathing-space-data-not-being-persisted`, for consistency with the PR title.

Reply:
@Daniel-Lam is this important or just a convention I should carry out going forward?

Reply:
@kuda1992 - not really that important, just that it looked a bit strange when it said "breathing data" 😄. Probably a good idea, nonetheless, to name the tag suffixes descriptively.